### DISCUSSION TITLE
Problems with PS2 “Call of Duty” games.

### DISCUSSION THREAD

**Discussion by RAMONCHITO85** [darthcloud/BlueRetro#662] [General]
Problems with PS2 “Call of Duty” games.

I've had my blueretro for a few months, but until today I haven't been able to play the PS2 Call of Duty games using it, especially games 2 and 3. Unlike other PS2 games, the control simply doesn't work. I did the latest update and it didn't work. Those are the only games I couldn't use a bluethoot controller.

Sorry for any writing errors. I'm from Brazil, and I used google translator to make this post.

**Comment by Trozzzul** [comment]
i too am having this issue on the latest version updated, specifically im using a ps4 controller and the specific game was CoD3 for ps2, the OP states multiple call of duty games dont work so id gladly test my actual other cod disc games, but the CoD3 i specifically used was being loaded from the hdd

**Comment by darthcloud** [comment]
I fixed the problem with CoD 2, 3 & Kung-Fu Panda (maybe other Activision games too?)

You can try the v1.9 beta here:
https://github.com/darthcloud/BlueRetro/actions/runs/6386376011

**Reply by DOOTGUY** [reply]
Which file to download?

**Reply by darthcloud** [reply]
take the HW1 one

**Reply by DOOTGUY** [reply]
Thanks

**Reply by DOOTGUY** [reply]
After update red and blue lights lit up together on my blueretro, and it's not working at all

**Reply by DOOTGUY** [reply]
Problem solved

**Reply by darthcloud** [reply]
Can't you describe the solution to the problem you had at first with the update. So that others know what to do?

**Reply by DOOTGUY** [reply]
I used NodeMCU-PyFlasher and adapter just bricked but then i used Flash Download Tool and then Blue Retro start workin (CoD & Kung Fu Panda works too tho)

**Reply by DOOTGUY** [reply]
what was the problem with these games?

**Reply by golimonkey** [reply]
I had the same issue with Bricked Blueretro (one from alliexpress). Flashing with NodeMCU-PyFlasher bricked the device (red and blue stay on). But flashing with Flash Download Tool fixed everything. Too bad that manual for the device recommends NodeMCU, there will be a lot of people with bricked receivers.

![AAA](https://github.com/darthcloud/BlueRetro/assets/71722655/9f84ef7a-5e17-405f-a951-7f548f1c5592)

**Reply by javadev-jef** [reply]
@golimonkey I have a Blueretro like this. Is yours working in CoD normally? And when you change games, your Blueretro disconnects the controller?

**Comment by DOOTGUY** [comment]
It's works perfectly fine
